src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$stopwatch-size: 80vmin;
+$center-size: 5vmin;
+$minutes-hand-hight: 20vmin + $center-size / 2;
+$minutes-hand-width: 3vmin;
+$seconds-hand-hight: 32vmin + $center-size / 2;
```

Comment:
Similarly, the variable `$seconds-hand-hight` appears to have a typo and should be corrected to `$seconds-hand-height`.